lib/vsc/install/shared_setup.py

```diff
@@ -144,13 +144,13 @@ def _log(self, level, msg, args):
 
 RELOAD_VSC_MODS = False
 
-VERSION = '0.9.8'
+VERSION = '0.9.9'
 
 log.info('This is (based on) vsc.install.shared_setup %s' % VERSION)
 
-# list of non-vsc packages that need python- prefix for correct rpm dependencies
+# list of non-vsc packages that do not need python- prefix for correct rpm dependencies
 # vsc packages should be handled with clusterbuildrpm
-PREFIX_PYTHON_BDIST_RPM = ('setuptools',)
+NO_PREFIX_PYTHON_BDIST_RPM = ()
```

Comment:
this looks confusing, you might want to change `()` to `tuple()` so it's clear what this is supposed to be?